digdag-standards/src/main/java/io/digdag/standards/command/kubernetes/KubernetesClientConfig.java

```diff
@@ -46,24 +51,49 @@ private static KubernetesClientConfig createFromSystemConfig(final Optional<Stri
         }
         final String keyPrefix = KUBERNETES_CLIENT_PARAMS_PREFIX + clusterName + ".";
         final Config extracted = validateParams(StorageManager.extractKeyPrefix(systemConfig, keyPrefix));
-        return create(clusterName,
-                extracted.get("master", String.class),
-                extracted.get("certs_ca_data", String.class),
-                extracted.get("oauth_token", String.class),
-                extracted.get("namespace", String.class));
+        if (extracted.get("use_kube_config", boolean.class, false)) {
+            io.fabric8.kubernetes.client.Config kubeConfig;
+            kubeConfig = getKubeConfigFromPath(extracted.get("kube_config_path", String.class));
```

Comment:
What do you think that 'validateParams' could be called for this 'kubeConfig' params? It means the impl. of the existing 'validateParams' could not be changed.

Reply:
Is it OK to "Check the value of` kubeConfig’s parameters` are null "?

Reply:
good point. they should be validated. up to you but one idea is to add new 'validateKubeConfig' method in this class. in this case,  we'd better to rename the existing 'validateParams' with 'validateConfig'. what do you think?

Reply:
> we'd better to rename the existing 'validateParams' with 'validateConfig'
I think so too, rename `validateParams` to `validateConfig`.

I also add `validateKubeConfig` method for kubeConfig’s.